app/utils/get-form.js

```diff
@@ -0,0 +1,43 @@
+const retrieveForm = async (id) => {
+  const response = await fetch(`/form-content/${id}`);
+  if (!response.ok) {
+    let error = new Error(response.statusText);
+    error.status = response.status;
+    throw error;
+  }
+
+  const form = await response.json();
+  if (!form.formTtl) throw new Error('Received form without formTtl');
+
+  return form;
+};
+
+// Split into definition and prefix because this structure is expected in instance components
```

Comment:
as the prefix is now part of the definition, wouldn't it make more sense to change the instance components?